labs/lab-2-javascript-game/script.js

```diff
@@ -0,0 +1,57 @@
+function startGame() {
+  const play = window.prompt("Do you want to play?");
+  if (play === "yes") {
+    var name = window.prompt("Enter your name:")
+    startCombat(name);
+  }
+}
+
+function getDamage() {
+  const min = 1;
+  const max = 5;
```

Comment:
Love this. You might even put min and max into the arguments -- then one getDamage function could generate lots of different damage ranges as needed.